internal/v2/remote_signer/remote_signer.go

```diff
@@ -0,0 +1,12 @@
+// Remote signer library that offloads private key operations to S2Av2.
+package remotesigner
+
+import (
+	"crypto"
+	"crypto/tls"
+)
+
+// NewRemoteSigner returns an instance of RemoteSigner, an implemenation of the crypto.Signer interface.
+func NewRemoteSigner(cert *tls.Certificate) crypto.Signer {
+	return cert.PrivateKey.(crypto.Signer)
```

Comment:
Can we add a TODO above the return statement to state that we will later be changing this to return an implementation of the `crypto.Signer` interface that calls S2A?